setup.py

```diff
@@ -23,7 +23,7 @@ def get_readme():
 
 setup(
     name='iodata',
-    version=get_version(),
+    version='0.1',
```

Comment:
See #11 for how to get `get_version` to work.

Reply:
Update the commit. Change the setting back